services/mailservers/fleet.go

```diff
@@ -231,5 +231,41 @@ func DefaultMailservers() []Mailserver {
 			Fleet:   params.FleetShardsTest,
 			Version: 2,
 		},
+		Mailserver{
```

Comment:
While I agree that hardcoding the storenodes here is bad, currently for Status we use an specific set of trusted storenodes that have high availability and store at least 30d of data. Using discv5 would mean that it is possible to discover store nodes that do not have these guarantees. And it's also problematic because the way status works right now, it will not request history for a period of time that it has already requested the history before, so if we choose a random storenode and we request history for some time interval, and that storenode has no data for that interval, then, the client will no longer have history for that period of time.

I wonder if storev3 sync solves this?